.github/workflows/ci.yml

```diff
@@ -42,8 +42,11 @@ jobs:
         os:
           - 'amazonlinux'
           - 'debian-9'
+          - 'debian-10'
           - 'ubuntu-1604'
+          - 'ubuntu-1804'
```

Comment:
Prob not for this one, but you might want to get 20.04 in here

Reply:
Done as well
Moved amazonlinux to amazonlinux-2